File: scala_day01/scala/com/oxysa/ifelse/Demo04.scala
package com.oxysa.ifelse

import scala.io.StdIn

/**
  * @author Yummy
  * @mail 873387751@qq.com
  * @date 2020-3-22 18:39
  *       练习1
  */
object Demo04 {
  def main(args: Array[String]): Unit = {
    /*
       声明2个int类型的变量并赋值，判断两个数之和，如果大于等于50，打印hello word
     */
    println("请输入第一个数值：")
    val num1: Int = StdIn.readInt()
    println("请输入第二个数值：")
    val num2: Int = StdIn.readInt()

    if (num1 + num2 >= 50) {
      println("hello word!")
    } else {
      println("条件不符合，exit!!")
    }


  }

}

File: day05/src/com/oxysa/outputstrem/Demo04.java
package com.oxysa.outputstrem;

import com.oxysa.streamtool.Tool;

import java.io.File;
import java.io.FileOutputStream;

/**
 * @author Yummy
 * @mail 873387751@qq.com
 * @date 2020-3-16 20:27
 * 案例：FileOutStream加入异常案例
 */
public class Demo04 {
    public static void main(String[] args) {
        FileOutputStream fos = null;
        try {
            //制造异常
            //System.out.println(1 / 0);
            //1创建FileOutStream对象 关联数据源文件
            fos = new FileOutputStream(new File("./day05/data/2.txt"), true);
            fos.write("hello".getBytes());
            System.out.println("写入成功");
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            Tool.outClose(fos);
            System.out.println("会执行嘛？？");
        }


    }
}
